services/email/views/owner.html

```diff
@@ -0,0 +1,5 @@
+<h1>Hello {{username}}!</h1>
+<p>You have been pertified as a hotel owner, your hotel is now available to other user.</p>
```

Comment:
maybe petrified xdd but i think we want to inform hotel owner that they have been VERIFIED not petrified XD